src/backend/web/graphql/queries.js

```diff
@@ -0,0 +1,99 @@
+const { gql } = require('apollo-server-express');
+
+const maxPostsPerPage = 30;
+
+module.exports.typeDefs = gql`
+  # 'Feed' matches our Feed type used with redis
+  type Feed {
+    name: String
+    url: String
+  }
+
+  # 'Post' matches our Post type used with redis
+  type Post {
+    author: String
+    title: String
+    html: String
+    text: String
+    published: String
```

Comment:
Should be `Date`, same for `updated` on next line.  If we can use the `Post` constructor somehow here vs pulling raw from Redis, we'll get it for free: https://github.com/Seneca-CDOT/telescope/blob/master/src/backend/post.js#L21-L22.